<launch>
    <rosparam command="load" file="$(find labrom_mav_control)/params/pelican.yaml"/>
    
    <node name="control_manager" pkg="labrom_mav_control" type="manager_node" output="screen">
      <!-- Remapping topics-->
      <!-- Remapping topics-->
      <remap from="odometry" to="/optical_flow/odometry"/>
      <remap from="imu" to="/fcu/imu"/>
      
      <remap from="trajectory" to="trajectory"/>   
      <remap from="cmd_attitude" to="cmd_attitude"/>
      <remap from="cmd_thrust" to="cmd_thrust"/>
      
      <!-- Loading parameters -->  
      <param name="mass" value="1.825"/>
      <param name="rate" value="20"/>
      <param name="take_off_accel" value="10"/>
      <param name="land_accel" value="9.2"/>
      <param name="climb_time" value="1.0"/>        
    </node>
</launch>
